src/main/java/uk/gov/pay/directdebit/common/exception/ApiExceptionMapper.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.pay.directdebit.common.exception;
+
+import com.google.common.collect.ImmutableMap;
+
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+public class ApiExceptionMapper implements ExceptionMapper<ApiException> {
+    /**
+     * Map an exception to a {@link Response}.
+     *
+     * @param exception the exception to map to a response.
+     * @return a response mapped from the supplied exception.
+     */
+    @Override
+    public Response toResponse(ApiException exception) {
+        ImmutableMap<String, String> entity = ImmutableMap.of("message", exception.getMessage());
+        return Response.status(exception.getStatusCode()).entity(entity).build();
```

Comment:
The exception mapper decides, depending on the exception, the status code to send back. So is better to use many exceptions named appropriately (so the stackTrace is displayed on logging and is clear what and where it happens)